libs/hdf-converters/src/utils/fingerprinting.ts

```diff
@@ -130,6 +132,11 @@ export function fingerprint(guessOptions: {
       guessOptions.data.indexOf('detailedreport') !== -1
     ) {
       return INPUT_TYPES.VERACODE;
+    } else if (
```

Comment:
not necessary to do raw text search for those fields when you can just add the name of those fields in the fingerprints definition starting line 26

Reply:
else if deleted